File: EPcar/src/ssr_pkg/src/messagePub_node.cpp
#include <ros/ros.h>
#include <std_msgs/String.h>

int main(int argc, char *argv[])
{
    ros::init(argc, argv, "messagePub_node");
    printf("Hello world!");

    ros::NodeHandle nh;
    ros::Publisher pub = nh.advertise<std_msgs::String>("qun",10);
    ros::Rate loop_rate(10);

    while(ros::ok())
    {
        printf("message!!!\n");
        std_msgs::String msg;
        msg.data = "message data";
        pub.publish(msg);
        loop_rate.sleep();
    }
    return 0;
}

File: EPcar/src/ssr_pkg/scripts/messagePub_node.py
#!/usr/bin/env python3

import rospy
from std_msgs.msg import String

if __name__ == "__main__":
    rospy.init_node("messagePub_node")
    rospy.logwarn("message")

    pub = rospy.Publisher("qun",String,queue_size=10)

    rate = rospy.Rate(2)

    while not rospy.is_shutdown():
        rospy.loginfo("message")
        msg = String()
        msg.data = "message_data"
        pub.publish(msg)
        rate.sleep()

    exit(0)